user/contactmethod/store.go

```diff
@@ -388,27 +333,32 @@ func (s *Store) FindMany(ctx context.Context, ids []string) ([]ContactMethod, er
 		return nil, err
 	}
 
-	rows, err := s.findMany.QueryContext(ctx, sqlutil.UUIDArray(ids))
+	uids := make([]uuid.UUID, len(ids))
+	for i, id := range ids {
+		uids[i] = uuid.MustParse(id)
+	}
```

Comment:
replace line 326 with:
`err, uids := validate.ParseManyUUID("ContactMethodID", ids, 50)`
and you can remove this loop